python/py-pyqt5-webengine/Portfile

```diff
@@ -61,6 +61,7 @@ if {${name} ne ${subport}} {
 
     destroot.cmd    make
     destroot.dir    ${build.dir}/build
+    destroot.target install
```

Comment:
this is likely prompted to get the py311 subport to build (sorry missed that issue)? It's likely caused by the fact that for Python 3.11 the option "python.pep517 yes" is set by default. I've seen that causing issues with other ports as well - we should probably investigate that and fix in the python PG instead of making changes to individual ports. 

So let's hold off with merging for a bit so I/we can take a look at that.